tests/test_XLSReader.py

```diff
@@ -43,7 +43,7 @@ def test_next_row(self):
             'Analysis Alias': 'GAE',
             'Sample Accession': None,
             'Sample ID': None,
-            'Sample Name': 'S1',
+            'Sample Name': '1',
```

Comment:
in `tests/resources/metadata.xlsx` I've only changed. the sample name from `S1` to `1`